bazel_tools/scala.bzl

```diff
@@ -381,9 +381,10 @@ rm -f {jar_output}
 def _scaladoc_jar_impl(ctx):
     # Detect an actual scala source file rather than a srcjar or other label
     srcFiles = [
-        src.path
-        for src in ctx.files.srcs
-        if src.is_source or src in ctx.files.generated_srcs
+# TODO: add a config param to omit the below
+#        src.path
+#        for src in ctx.files.srcs
+#        if src.is_source or src in ctx.files.generated_srcs
     ]
```

Comment:
We need these Scaladocs when we publish the real thing.

Perhaps you can tell `daml-sdk-head` to run `bazel build` with the parameter `--build_tag_filters '-scaladoc,-pdfdocs'` instead?

Reply:
That’s not going to work, there are two things you need to change:
1. The Haskell release script needs to gain an option to disable scaladoc (and probably also javadoc) builds.
2. daml-sdk-head then needs to pass this option through either optionally or always (I’m fine with both).

Reply:
@SamirTalwar-DA 
Yeah, it's solely about the dev feedback loop.

Thanks, I might need to take a deeper look, as both:
- `export BAZEL_MODE_FLAGS="--build_tag_filters '-scaladoc,-pdfdocs'" && daml-sdk-head`
- `bazel build ${BAZEL_MODE_FLAGS[@]:-} --build_tag_filters '-scaladoc,-pdfdocs' //release:sdk-release-tarball-ce.tar.gz`

didn't work (still build the scaladocs).

Reply:
Tags aren’t going to get you anywhere here. The haskell script selects individual targets that it depends on. You need to change that and not go via tags.